readthedocs/embed/v3/views.py

```diff
@@ -8,23 +8,37 @@
 import structlog
 from django.conf import settings
 from django.core.cache import cache
-from django.shortcuts import get_object_or_404
 from rest_framework import status
 from rest_framework.renderers import BrowsableAPIRenderer, JSONRenderer
 from rest_framework.response import Response
 from rest_framework.views import APIView
 from selectolax.parser import HTMLParser
 
 from readthedocs.api.mixins import CDNCacheTagsMixin, EmbedAPIMixin
+from readthedocs.api.v2.permissions import IsAuthorizedToViewVersion
 from readthedocs.api.v3.permissions import HasEmbedAPIAccess
 from readthedocs.core.utils.extend import SettingsOverrideObject
 from readthedocs.embed.utils import clean_references
-from readthedocs.projects.constants import PUBLIC
 from readthedocs.storage import build_media_storage
 
 log = structlog.get_logger(__name__)
 
 
+class IsAuthorizedToGetContenFromVersion(IsAuthorizedToViewVersion):
+
+    """
+    Checks if the user from the request has permissions to get content from the version.
+
+    If the URL is from an external project, we return ``True``,
+    since we don't have a project to check for.
```

Comment:
Does this mean that all PR builds are authorized? Or is there another mechanism that catches this? Would be nice to add that in the explanation.

Reply:
As mentioned in https://github.com/readthedocs/readthedocs.org/pull/10261#discussion_r1181957833, here external means an external site.

I have replaced `external project` with `external site`, but let me know if there is anything else we can add here to make it more clear.

Reply:
Thanks. It's because "version" is already mentioned in the context, so I was confused about what kind of "external" we were talking about.

I think it's great this way, it's of course unfortunate when terminology overlaps, but this was a good solution :+1: